### PR TITLE
RN 41026

### PR DESCRIPTION
**Pull request by EdithLansens**
@bertbsky, could you please review this pull request to confirm the accurate inclusion of release note 41026 in the user guide? Thanks in advance!

### CODE REVIEW

File: user-guide/Getting_started/The_DataMiner_User_Interface/DataMiner_Cube/Computer_settings.md [hunk 1]
```diff
@@ -104,6 +104,10 @@ On the main *Advanced* page, the following settings are available:
   > [!NOTE]
   > Enabling this setting involves a potential security risk: If multiple DataMiner users share the same Windows account on a particular computer, they will also share the same browser cache and, as a consequence, the same authentication on third-party websites.
 
+- **Clear browser cache on signout**: Available from DataMiner 10.3.0 [CU21]/10.4.0 [CU9]/10.4.12 onwards<!--RN 41026-->.  Select this checkbox to clear the Edge WebView2 browser cache upon signing out. This ensures that the SAML authentication window will appear the next time you log on. We strongly recommend enabling this setting on multi-user computers to allow different users to sign in when SAML authentication is used.
```
Comment (bertbsky) [inline user-guide/Getting_started/The_DataMiner_User_Interface/DataMiner_Cube/Computer_settings.md:107]: As we have some more room here than in the Cube UI, I would reword the following:
> We strongly recommend enabling this setting on **multi-user computers** to allow...

to
> We strongly recommend enabling this setting on **computers where different DataMiner users share the same Windows user desktop session** to allow...

Comment (EdithLansens) [reply user-guide/Getting_started/The_DataMiner_User_Interface/DataMiner_Cube/Computer_settings.md:107]:
```suggestion
- **Clear browser cache on signout**: Available from DataMiner 10.3.0 [CU21]/10.4.0 [CU9]/10.4.12 onwards<!--RN 41026-->.  Select this checkbox to clear the Edge WebView2 browser cache upon signing out. This ensures that the SAML authentication window will appear the next time you log on. We strongly recommend enabling this setting on computers where different DataMiner users share the same Windows user desktop session to allow these users to sign in when SAML authentication is used.
```

Done!

**Review by MariekeGO — APPROVED** [review]
